pkg/util/testing/wrapper.go

```diff
@@ -591,6 +591,80 @@ type ClusterTrainingRuntimeWrapper struct {
 	trainer.ClusterTrainingRuntime
 }
 
+func MakeTrainingRuntimeWithoutModelInitializerWrapper(namespace, name string) *TrainingRuntimeWrapper {
```

Comment:
Rather than creating another wrapper function, can we re-use `MakeTrainingRuntimeWrapper`, and just remove the  `model-initializer` container in the integration test ?